charts/community/ai-lab/chatbot/OWNERS

```diff
@@ -0,0 +1,15 @@
+chart:
+  name: chatbot-ai-software-template
```

Comment:
Should we call this chatbot-ai-sample? Software template is a RHDH specific term.

Reply:
Addressed in [4b440cf](https://github.com/openshift-helm-charts/charts/pull/1669/commits/4b440cf74bdb485cfffdca12d1358e72d661e29a)

Reply:
I'll also update the helm chart package to be consistent with the name in the `Chart.yaml`